File: PropertyDimashAngular/Models/Flat.cs
namespace PropertyDimashAngular.Models
{
    public class Flat : Property
    {
        public int? PropertyFloor { get; set; }
        public int? TotalFloor { get; set; }
        public bool? FormerDormitory { get; set; }
        public string? Balcony { get; set; }
        public bool? BalconyGlazed { get; set; }
        public string? Door { get; set; }
        public string? Parking { get; set; }
        public string? PropertyFurnished { get; set; }
        public string? Floor { get; set; }
        public string ResidentialComplexId { get; set; }
        public ResidentialComplex? ResidentialComplex { get; set; }
    }
}

File: PropertyDimashAngular/ClientApp/src/app/property/flat.ts
import { ResidentialComplex } from "../residentialComplex/residential-complex";
import { Property } from "./property";

export interface Flat {
  propertyFloor: number;
  totalFloor: number;
  formerDormitory: boolean;
  balcony: string;
  balconyGlazed: boolean;
  door: string;
  parking: string;
  propertyFurnished: string;
  floor: string;
  residentialComplexName: string;
  residentialComplex: ResidentialComplex
}
